src/main/views/teamDropdownView.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {BrowserView, BrowserWindow, ipcMain, IpcMainEvent} from 'electron';
+import {CombinedConfig, Team} from 'types/config';
+
+import {
+    CLOSE_TEAMS_DROPDOWN,
+    EMIT_CONFIGURATION,
+    OPEN_TEAMS_DROPDOWN,
+    UPDATE_TEAMS_DROPDOWN,
+    UPDATE_DROPDOWN_MENTIONS,
+    REQUEST_TEAMS_DROPDOWN_INFO,
+    RECEIVE_DROPDOWN_MENU_SIZE,
+    SET_SERVER_KEY,
+} from 'common/communication';
+import * as AppState from '../appState';
+import {TAB_BAR_HEIGHT, THREE_DOT_MENU_WIDTH, THREE_DOT_MENU_WIDTH_MAC, MENU_SHADOW_WIDTH} from 'common/utils/constants';
+import {getLocalPreload, getLocalURLString} from 'main/utils';
+import * as WindowManager from '../windows/windowManager';
+
+export default class TeamDropdownView {
```

Comment:
We still use "Team" in the desktop app to denote a server, even though it's wrong :P
I just did this for consistency. @Willyfrog do we do a mass rename here?
I feel like it might be worth doing this as a cleanup step en masse later on.

Reply:
I'm always in the verge of doing it, so maybe it is the right time to do it :)

Reply:
0/5 on doing it now or later but something we should do we eventually